### PR TITLE
Modal Body is done.

### PR DESCRIPTION
**Pull request by CotyJ**
Most new code is styling or the new modal. The rest is CSS tweaks and deleted comments mostly

### CODE REVIEW

File: client/src/components/Ratings/ReviewModal.jsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            </div>
+            <div className="review-desc-right">
+              Premium
+            </div>
+          </div>
+        </div>
+
+        <div className="review-modal-selector-container">
+          <div> Review Summary </div>
+          <input placeholder="  I really loved this product!" style={{ width: '100%', border: '1px solid black' }} />
```
Comment (Dalphus) [inline client/src/components/Ratings/ReviewModal.jsx:106]: input tags should have a `name` attribute

File: client/src/components/Ratings/ReviewsList.jsx [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
+  const [openModal, setOpenModal] = useState(false);
 
   function handleOptionChange(event) {
     console.log(event.target.value);
     sort(event.target.value);
   }
 
-  function addReviewModal() {
-    console.log("ADD A REVIEW BRO");
-  }
```
Comment (Dalphus) [inline client/src/components/Ratings/ReviewsList.jsx:15]: giving your WIP functions a placeholder return statement makes them complete enough to put onto the main branch.
A function that returns something can have a test written for it too, even if that test is just `result !== undefined`

File: client/src/components/Ratings/ReviewsList.jsx [hunk 3]
```diff
@@ -68,11 +68,14 @@ function ReviewsList({ results, sort }) {
...
         className='review-button'
         type="button"
-        onClick={() => addReviewModal()}
+        onClick={() => { setOpenModal(true); }}
+        style={{ float: 'left' }}
       >
         Add A Review +
       </button>
 
+      {openModal && <ReviewModal closeModal={setOpenModal} />}
```
Comment (Dalphus) [inline client/src/components/Ratings/ReviewsList.jsx:77]: while cool, this syntax hurts readability. Use a ternary operator or an IIFE

File: client/src/components/Ratings/index.jsx [hunk 4]
```diff
@@ -27,23 +20,29 @@ function getPercentageRecommended(results) {
...
+  }
+
+  return "";
 }
 
+
 // WIP to filter by rating
 function filterByRating(stars) {
   console.log(`You clicked to filter ${stars} stars!`);
 }
```
Comment (Dalphus) [inline client/src/components/Ratings/index.jsx:43]: remove WIP functions or make them return something
also remove console logs in your final commit

File: client/src/components/Ratings/index.jsx [hunk 5]
```diff
@@ -80,105 +78,33 @@ function addStarAndBar(results, stars) {
...
 
-      {/* Use generateStars function from Overview */}
       <div>Ratings And Reviews</div>
 
       <div className="review-summary">
 
         <h2 id="big-review-num">
-          <div style={{ float: 'left' }}>{averageToNearestTenth(results) ? averageToNearestTenth(results) : "Loading..."}</div>
-          <div id="review-stars"><Stars rating={averageToNearestTenth(results)} /></div>
+          <div style={{ float: 'left' }}>{averageToNearestTenth(results) ? averageToNearestTenth(results) : ""}</div>
```
Comment (Dalphus) [inline client/src/components/Ratings/index.jsx:98]: this line should be broken up into several, it is difficult to read

**Review by Dalphus — CHANGES_REQUESTED** [review]
mfw you use inline styles
image

**Review by BlandSchwad — APPROVED** [review]

**Review by Dalphus — APPROVED** [review]

**Pull request merged**